cmd/ttn-lw-cli/commands/root.go

```diff
@@ -207,6 +209,35 @@ func optionalAuth() error {
 	return nil
 }
 
+func confirmChoice(warning string) bool {
```

Comment:
We probably need to have a non-interactive option here too, i.e. with a `--force` flag or something